### PR TITLE
chore: fix Xcode 13 warnings SQPIT-663

### PR DESCRIPTION
**Pull request by billypchan**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQPIT-663" title="SQPIT-663" target="_blank">TaskSQPIT-663</a>  Xcode 13.1 upgrade
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Apply autofix by Xcode 13 for the warnings

### CODE REVIEW

File: WireTransport.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1213,6 +1214,7 @@
 			baseConfigurationReference = 547712121B751F7900223568 /* project.xcconfig */;
 			buildSettings = {
 				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
```
Comment (billypchan) [inline WireTransport.xcodeproj/project.pbxproj:1217]:
```suggestion
```

**Comment by billypchan** [comment]
release patch

File: Resources/Configurations/version.xcconfig [hunk 2]
```diff
@@ -1,2 +1,2 @@
 MAJOR_VERSION = 84
-CURRENT_PROJECT_VERSION = 84.0.2
+CURRENT_PROJECT_VERSION = 84.0.3
```
Comment (billypchan) [inline Resources/Configurations/version.xcconfig:2]:
```suggestion
CURRENT_PROJECT_VERSION = 84.0.2
```

**Comment by billypchan** [comment]
release patch

**Comment by billypchan** [comment]
release patch

**Comment by zenkins** [comment]
Hello, @billypchan! Merging PR for command **release patch** for PR #**235**. Will create a **patch** release after merged.
Next version is 84.0.3.

**Pull request merged**